lib/pa11y-ci.js

```diff
@@ -113,6 +148,9 @@ function pa11yCi(urls, options) {
 			// results to the report object
 			try {
 				const results = await pa11y(url, config);
+				await cycleReporters(reporters, 'results', results, {config,
+					url,
+					urls});
 				processResults(results, config, url);
 			} catch (error) {
```

Comment:
Propose adding an `error` method, which the `pa11y` reporters implement:

```suggestion
			} catch (error) {
				await cycleReporters(reporters, 'error', error, {config,
					url,
					urls});
```